flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -74,6 +75,7 @@
  * are either compatible or requiring state migration after restoring the state backends.
  */
 @SuppressWarnings("serial")
+@ExtendWith(ParameterizedTestExtension.class)
 public abstract class StateBackendMigrationTestBase<B extends StateBackend> extends TestLogger {
```

Comment:
`TestLogger` is a JUnit4 rule/extension
Here should be used `META-INF/services/org.junit.jupiter.api.extension.Extension`